spec/system/visit_msa_component_edit_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'Edit MSA Component Page', type: :system do
+  before(:each) do
+    login_user
+  end
+
+  let(:msa_component_1) { create(:msa_component) }
+  let(:team1) { NewTeamEvent.create(name: 'team 1').team }
+  let(:create_teams) { team1 }
+
+  context 'when team option is selected' do
+    it 'successfully associates team id with component' do
+      create_teams
+      visit edit_msa_component_path(msa_component_1)
+      select(team1.name, from: 'component[team_id]')
+      click_button 'Associate Team'
```

Comment:
btw, you **can** also use localisation in the tests, so you don't have to update tests when you change the content.

Reply:
This is a very good idea if I can get it to work